<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>186</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Radio</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/antenna.svg</normaloff>:/antenna.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Nombre:</string>
     </property>
     <property name="buddy">
      <cstring>name</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="name"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&amp;NÃºmero:</string>
     </property>
     <property name="buddy">
      <cstring>number</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="number"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancelar</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/delete.svg</normaloff>:/delete.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Aceptar</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/ok.svg</normaloff>:/ok.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
